WSDiscovery.py

```diff
@@ -405,9 +405,9 @@ def addElementWithText(doc, parent, name, ns, value):
 
 def getDocAsString(doc):
     outStr = ""
-    stream = StringIO.StringIO(outStr)
+    stream = io.StringIO(outStr)
     stream.write(doc.toprettyxml())
-    return stream.getvalue()
+    return stream.getvalue().encode('UTF-8')
```

Comment:
This is actually returning a bytes object now, not unicode as in the original Python2 getDocAsString.